typescript/src/templater.ts

```diff
@@ -68,7 +69,7 @@ function processOneTemplate(
     for (const service of api.services) {
       result.push(
         renderFile(
-          outputFilename.replace(/\$service/, service.name!.toLowerCase()),
+          outputFilename.replace(/\$service/, service.name!.toSnakeCase()),
```

Comment:
There must be a change in the baseline, the place where we import the renamed JSON file.

Reply:
got it, thanks!